### PR TITLE
surface data for timeConfig

### PR DESCRIPTION
**Pull request by 0xlarry**

### CODE REVIEW

File: src/types/api.ts [hunk 1]
```diff
@@ -334,18 +344,20 @@ type OfferTensorBuyAction = {
...
   eventConfig: NftBidBuy;
+  timeConfig?: TimeConfig;
 };
 
 /**
  * Full bounty step requirement type.
  */
 export type ApiRequirement =
-  | OfferSwapAction
+  | (OfferSwapAction & { timeConfig: TimeConfig })
```
Comment (nardevvv) [inline src/types/api.ts:360]: Is this required or should it be optional?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "displayName": "Torque TypeScript SDK",
   "name": "@torque-labs/torque-ts-sdk",
   "author": "Torque Labs",
-  "version": "0.0.66",
+  "version": "0.0.67",
```
Comment (nardevvv) [inline package.json:5]: Incorrect version. Pull main here. Current version is "0.0.71"

File: src/types/api.ts (hunk 1 above)
Comment (0xlarry) [reply src/types/api.ts:354]: removed, set to optional above.

File: package.json (hunk 2 above)
Comment (0xlarry) [reply package.json]: done

**Pull request merged**